src/lib/Libwin/win_remote_shell.c

```diff
@@ -45,6 +45,7 @@
 #include <pbs_config.h>   /* the master config generated by configure */
 #include <pbs_version.h>
 #include "win.h"
+#include "log.h"
 #include "win_remote_shell.h"
 
 
```

Comment:
Please add error logging in https://github.com/PBSPro/pbspro/blob/bb1fa7be4315449eb94bb63e280b260c4cec3c0f/src/lib/Libwin/win_remote_shell.c#L99